scripts/cubemx-importer.stanza

```diff
@@ -305,6 +314,20 @@ defn dac-pin? (s:String) -> True|False :
 defn dac-pins (pins:Tuple<Stm32Pin>) -> Tuple<DacOption>:
   to-tuple(seq(DacOption, pins-with-signal(pins, dac-pin?)))
 
+; Only generates support statements for timer pins that have
+; format TIM[0-9]+_CH[0-9]+ - example TIM3_CH1
+; It ignores other types (exhaustive list of formats)
+; TIM15_BK, TIM15_BKIN, TIM15_CH1N, TIM19_ETR, TIM1_BK2,
+; TIM1_BKIN2, TIM1_BKIN2_COMP1, TIM1_BKIN_COMP1, TIMX_IC1
+defn timer-pin? (s:String) -> True|False :
+  val b = not empty?(regex-match("^TIM[0-9]+_CH[0-9]+[^N]", s))
+  ;println("%_ : %_" % [s, b])
```

Comment:
we can use

```
not empty?(regex-match("^TIM[0-9]+_CH[0-9]+[^N]", s))
```

here w/o the print debugging

Reply:
I have tested this file on 6 files. I need to test them on another 1200+ files. So I would like to keep this debug statement until I test against more of the files

Reply:
Then we should wait until it's fully tested before merging, we really shouldn't be putting print debugging into production.